example/src/App.tsx

```diff
@@ -1,9 +1,16 @@
 import React from 'react'
-import { Main } from '@graasp/ui'
+import { Main, MainMenu, MenuItem } from '@graasp/ui'
+import HomeIcon from '@material-ui/icons/Home'
 import '@graasp/ui/dist/index.css'
 
 const App = () => {
-  return <Main />
+  const mainMenu = (
+    <MainMenu>
+      <MenuItem text='Home' icon={<HomeIcon />} selected={true}></MenuItem>
```

Comment:
Factor out (and translate) "Home"?

Reply:
Do we really need to factor out and translate? This file is in `example`, but we can include translation if you think it is necessary (but for now the language cannot be changed in anyway).

Reply:
@pyphilia, no, that's OK. I missed that this was inside the example folder.